[2024-03-29T01:53:47.433+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509
[2024-03-29T01:53:47.435+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task'
[2024-03-29T01:53:47.494+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509
[2024-03-29T01:53:47.495+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task'
[2024-03-29T01:53:47.550+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_kubernetes_executor.start_task manual__2024-03-29T01:53:37.362760+00:00 [queued]>
[2024-03-29T01:53:47.567+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_kubernetes_executor.start_task manual__2024-03-29T01:53:37.362760+00:00 [queued]>
[2024-03-29T01:53:47.569+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-03-29T01:53:47.601+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2024-03-29 01:53:37.362760+00:00
[2024-03-29T01:53:47.617+0000] {standard_task_runner.py:57} INFO - Started process 100 to run task
[2024-03-29T01:53:47.622+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_kubernetes_executor', 'start_task', 'manual__2024-03-29T01:53:37.362760+00:00', '--job-id', '5', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_kubernetes_executor.py', '--cfg-path', '/tmp/tmp4wora495']
[2024-03-29T01:53:47.636+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask start_task
[2024-03-29T01:53:47.729+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509
[2024-03-29T01:53:47.730+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_kubernetes_executor/run_id=manual__2024-03-29T01:53:37.362760+00:00/task_id=start_task'
[2024-03-29T01:53:47.733+0000] {task_command.py:415} INFO - Running <TaskInstance: example_kubernetes_executor.start_task manual__2024-03-29T01:53:37.362760+00:00 [running]> on host 1279dfb12ee8
[2024-03-29T01:53:47.839+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_kubernetes_executor' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T01:53:37.362760+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T01:53:37.362760+00:00'
[2024-03-29T01:53:47.846+0000] {logging_mixin.py:151} INFO - annotated!
[2024-03-29T01:53:47.846+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-03-29T01:53:47.857+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=example_kubernetes_executor, task_id=start_task, execution_date=20240329T015337, start_date=20240329T015347, end_date=20240329T015347
[2024-03-29T01:53:47.881+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-03-29T01:53:47.934+0000] {taskinstance.py:2776} INFO - 3 downstream tasks scheduled from follow-on schedule check
